driver/Cargo.toml

```diff
@@ -12,6 +12,7 @@ license.workspace = true
 ce-core.workspace = true
 ce-shell.workspace = true
 color-eyre.workspace = true
+dunce = { workspace = true }
```

Comment:
Here as well

Reply:
Yes, I've now addressed this in the latest commit 🙂